content/en/getting_started/synthetics/private_location.md

```diff
@@ -42,7 +42,10 @@ Your private locations test results display identically to your managed location
 
 ## Create your private location
 
-1. Install [Docker][4] on a machine. To get started quickly, you can install Docker on a virtual machine such as [Vagrant Ubuntu 16.04][2].
+1. You can use any Unix virtual machine with Docker or without a virtual machine. Install [Docker][4] on a machine, or use another container runtime such as [Podman][10].
```

Comment:
```suggestion
1. Install [Docker][4] on a Unix-like machine, or use another container runtime such as [Podman][10].
```
I don't understand the "or without a virtual machine" bit, so this suggestion might change the meaning of the sentence.

Reply:
@hestonhoffman I think this makes sense! I feel documentation wise nothing could beat the Puppet documentation 😉